junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/Root.java

```diff
@@ -33,6 +36,12 @@
 	private final Map<TestEngine, TestDescriptor> testEngineDescriptors = new LinkedHashMap<>(4);
 	private final ConfigurationParameters configurationParameters;
 
+	/**
+	 * Maps to store list of {@link TestDescriptor} which are excluded.
+	 * key: Reason of exclusion.
+	 */
+	private final Map<String, List<TestDescriptor>> exclusionReasonToDescriptorMap = new LinkedHashMap<>();
```

Comment:
I think that if you rename the map to something like `excludedTestDescriptorsByReason`, you could get rid of the comment. Also, does it really need to be a `LinkedHashMap`?

Reply:
Will Rename It, It doesn't matter which map implementation we use, ordered is always better. :)